changeset:   780:717f918a0e00
tag:         tip
user:        YangZh@MCW-yZhang.MCW-CC.CN
date:        Wed May 28 20:21:52 2014 +0800
files:       Impress/libgltf/src/FPSCounter.cpp
description:
1) remove "Render Total Time" string from it
2) move the FPS text to the bottom right corner
3) reduce the font size to 1/3 of what it now


diff -r 3ee778f06e4e -r 717f918a0e00 Impress/libgltf/src/FPSCounter.cpp
--- libgltf/src/FPSCounter.cpp	Wed May 28 18:00:06 2014 +0800
+++ libgltf/src/FPSCounter.cpp	Wed May 28 20:21:52 2014 +0800
@@ -110,11 +110,7 @@
         }
     }
 
-    pFont->printFormattedString(20, pViewport->height - 30, 20, "FPS: %d",
-                                mFPS);
-    pFont->printFormattedString(20, pViewport->height - 60, 20,
-                                "Render Total Time: %.6f ms",
-                                mTotalTime * 1000);
+    pFont->printFormattedString(pViewport->width - 100, 30, 15, "FPS: %d",mFPS);
     glEnable(GL_DEPTH_TEST);
 }
 #endif //ENABLE_FPS

